zulipterminal/ui_tools/views.py

```diff
@@ -747,8 +747,10 @@ def topics_view(self, stream_button: Any) -> Any:
         return w
 
     def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
-        if (is_command_key('SEARCH_STREAMS', key) or
-                is_command_key('SEARCH_TOPICS', key)):
+        if (
+            is_command_key('SEARCH_STREAMS', key)
+            or is_command_key('SEARCH_TOPICS', key)
+           ):
```

Comment:
Is the bracket change supposed to happen?

Reply:
Same as the other 'alternative' in-line comment.

Reply:
This is the same feedback as from @preetmishra ?

Reply:
See my comment below re continuation character vs parentheses.

Also possible alternative style with iterable passed to `is_command_key`... and how flake8 missed some things due to the presence of continuation characters!